### PR TITLE
Merge Navigation Section Structure and Content

### PR DESCRIPTION
**Pull request by samuelsogbesan**
- Update HTML structure to contain necessary structure for Navigation page. The page contains a grid of 4 cards.
- Add CSS rules to style and format cards into a 2x2 CSSGrid, referencing the colour variables designed in root for layout and themeing.

Here is a visual prompt for the final design:
image

This is the page in its current state:
image

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -45,6 +25,46 @@ body {
...
+  display:grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows:1fr 1fr;
+  color:white;
+}
+
+.navigation-item {
+    display:flex;
+    flex-direction:column;
+    align-items:center;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: [Alphabetize declarations.](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order)

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -45,6 +25,46 @@ body {
...
+#navigation {
+  font-size: 5vw;
+  display:grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows:1fr 1fr;
+  color:white;
+}
+
+.navigation-item {
+    display:flex;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: [add a space after `:`](https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops), here and elsewhere

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -45,6 +25,46 @@ body {
...
+    background: white;
+    transition: 0.5s filter ease-in-out;
+    margin:0;
+}
+
+.navigation-item:hover {
+  filter:invert(1);
+}
+
+.navigation-item[data-link="introduction"]{
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: [use single quotation marks](https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks), here and below

**Review by krasimirgg — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Resolved in commit ab25cfb

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Quotes changed with commit 1840168

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Attributes alphabetised in commit ab8c4a6

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Further formatting in commit 7a710ce

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -45,6 +25,46 @@ body {
   max-width: 100%;
 }
 
+#navigation {
+  font-size: 5vw;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  color: white;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: [Alphabetize declarations.](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order)

**Review by krasimirgg — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/style.css (hunk 4 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Updated with commit 436779b

Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Updated in commit 436779b

**Review by krasimirgg — APPROVED** [review]
LGTM!

I like this design!

**Review by krasimirgg — APPROVED** [review]
LGTM

**Pull request merged**